client/blocks/comments/comment-actions.jsx

```diff
@@ -80,8 +78,4 @@ const CommentActions = ( {
 	);
 };
 
-CommentActions.defaultProps = {
-	onReadMore: noop,
```

Comment:
This noop can be safely removed because 1) `Button` doesn't mind when `onClick` prop is missing; 2) the components has only one usage (`<PostComment>`) which always passes a valid callback.